sources/packages/backend/apps/db-migrations/src/sql/SFASIndividuals/Update-swap-csl-bcsl-columns.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE
```

Comment:
If you do this, you're going to lose csl_overaward values, won't you?

Reply:
We've talked about maybe renaming the columns as a safe way to not lose data. I'm not sure this would work as the values from csl_overaward will be updated with bcsl_overaward and then bcsl_overaward won't be update as the values are the same.

Reply:
I saw that you left an image showing that there's no loss and that worked. Thanks for the syncup.